[test-triage] chip_sw_coremark

### Hierarchy of regression failure

Chip Level

### Failure Description

```
ERROR: Skipping '//sw/device/benchmarks/coremark/coremark_top_earlgrey:1_sim_dv': no such package 'sw/device/benchmarks/coremark/coremark_top_earlgrey': BUILD file not found in any of the following directories. Add a BUILD file to a directory to mark it as a package.
   - /workspace/mnt/repo_top/sw/device/benchmarks/coremark/coremark_top_earlgrey
  WARNING: Target pattern parsing failed.
  ERROR: no such package 'sw/device/benchmarks/coremark/coremark_top_earlgrey': BUILD file not found in any of the following directories. Add a BUILD file to a directory to mark it as a package.
   - /workspace/mnt/repo_top/sw/device/benchmarks/coremark/coremark_top_earlgrey
  INFO: Elapsed time: 19.608s
  INFO: 0 processes.
  FAILED: Build did NOT complete successfully (0 packages loaded)
  FAILED: Build did NOT complete successfully (0 packages loaded)
  make: *** [/workspace/mnt/repo_top/hw/dv/tools/dvsim/sim.mk:73: sw_build] Error 1
```


### Steps to Reproduce

Observed in all nightlies since at least 12 August.
[cab5ffb90](https://github.com/lowrisc/opentitan/tree/cab5ffb9055258dcad355d60ac2c655519fbc000)
Build seed 3773991536, VCS

### Tests with similar or related failures

NA

--- a/hw/top_earlgrey/dv/chip_sim_cfg.hjson
+++ b/hw/top_earlgrey/dv/chip_sim_cfg.hjson
@@ -1257,7 +1257,7 @@
     {
       name: chip_sw_coremark
       uvm_test_seq: chip_sw_base_vseq
-      sw_images: ["//sw/device/benchmarks/coremark/coremark_top_earlgrey:1:external"]
+      sw_images: ["//third_party/coremark/top_earlgrey:coremark_test:1"]
       en_run_modes: ["sw_test_mode_test_rom"]
       run_opts: ["+en_uart_logger=1",
                  "+sw_test_timeout_ns=22_000_000"]

